MonticelloTonel-Core.package/TonelWriter.class/instance/methodDefinitionOf..st

```diff
@@ -1,6 +1,8 @@
 private definitions
 methodDefinitionOf: aMethodDefinition
 	^ self toSTON: (OrderedDictionary new 
-		at: #category put: aMethodDefinition category; 
+		at: #category put: aMethodDefinition category;
+		in: [:dict | aMethodDefinition timeStamp ifNotNil: [:stamp |
```

Comment:
I think `in:` is frowned upon by core Squeak developers, when a temporary variable will do as well.

Reply:
It's minimally invasive. Whether this is appropriate or not depends on how you will decide about moving foreward.